src/mbgl/gl/index_buffer.hpp

```diff
@@ -14,6 +14,10 @@ class IndexVector {
 public:
     static constexpr std::size_t groupSize = DrawMode::bufferGroupSize;
 
+    void reserve(std::size_t size) {
+        v.reserve(size);
```

Comment:
Since you're using `reserve` as if it reserves a number of _index groups_ rather than a number of _indexes_, this should be `size * groupSize`.